* If there is an exception of type int and float in one TRY block then we can have MULTIPLE CATCH BLOCKS which
can take the thrown values as per the datatype

try
{

}

catch(int e)
{
}

catch(float e)
{
}

catch(MyException e)
{
}

* If we don't know what type of exception it is then we can have a CATCH ALL BLOCK WHICH IS ALSO CALLED ELLIPSE

Ellipse is defined as below

catch(...) // three dots
{
}

* Hence for the datatype for which the catch block is defined the thrown values will go to the corresponding catch
blocks. For all other datatypes thrown CATCH ALL will catch it.

* If we are using CATCH ALL block then we are not giving proper message to the user

* ALSO NOTE THAT CATCH ALL BLOCKS MUST BE WRITTEN LAST AFTER ALL THE CATCH BLOCKS

* If Catch All block is written first then all the exception will be handled by it only and no other catch
block after it will be executed



